.github/workflows/build-ios.yml

```diff
@@ -30,7 +30,11 @@ jobs:
         uses: actions/checkout@v4
         with:
           path: 'cbreez'
-
+      - name: Install rust
+        run: |
+          rustup set auto-self-update disable
+          rustup toolchain install stable --profile minimal
+          
```

Comment:
Is this not needed on `Build Android` workflow?

Reply:
Added that.